src/content/en/tools/lighthouse/audits/aspect-ratio.md

```diff
@@ -0,0 +1,48 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Displays Images With Incorrect Aspect Ratio" Lighthouse audit.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_blink_components: N/A #}
+
+# Displays Images With Incorrect Aspect Ratio  {: .page-title }
+
+## Overview {: #overview }
+
+If a rendered image has a significantly different aspect ratio from the aspect ratio in its
+source file (the "natural" aspect ratio), then the rendered image may look distorted, possibly
+creating an unpleasant user experience.
+
+## Recommendations {: #recommendations }
+
+Use `vw` for *both* the width and height of images.
```

Comment:
Hm not sure about the focus on `vw` here since we'd be perfectly happy here if people used no explicit size at all, any fixed sizes that were the same ratio`img.half-size { width: var(--real-width) / 2; height: var(--real-height) / 2; }`, or [object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit).

Maybe it would be better to just say to avoid setting width and height as % of a variably-sized container?

Reply:
SGTM! Yeah I was just winging it and couldn't find a good general rule...